src/main/java/embl/ebi/variation/eva/pipeline/steps/tasklet/IndexesCreate.java

```diff
@@ -50,14 +49,9 @@
     public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
 
         ObjectMap pipelineOptions = variantJobsArgs.getPipelineOptions();
-        if (pipelineOptions.getBoolean(SKIP_INITIALIZE_INDICES)) {
-            logger.info("skipping indices creation step, initialize.indices.skip is set to {} ",
-                    pipelineOptions.getBoolean(SKIP_INITIALIZE_INDICES));
-        } else {
-            MongoOperations operations = MongoDBHelper.getMongoOperationsFromPipelineOptions(pipelineOptions);
-            operations.getCollection(pipelineOptions.getString("db.collections.features.name"))
-                    .createIndex(new BasicDBObject("name", 1), new BasicDBObject("sparse", true));
-        }
+        MongoOperations operations = MongoDBHelper.getMongoOperationsFromPipelineOptions(pipelineOptions);
+        operations.getCollection(pipelineOptions.getString("db.collections.features.name"))
+                .createIndex(new BasicDBObject("name", 1), new BasicDBObject("sparse", true).append("background", true));
```

Comment:
I'm not sure why, but `mongoOperations.getCollection` returns a `DBCollection` (instead of a `MongoCollection`) which [has a different prototype](http://api.mongodb.com/java/3.0/com/mongodb/DBCollection.html#createIndex-com.mongodb.DBObject-com.mongodb.DBObject-), and sparse and background are passed as options.